src/components/ha-selector/ha-selector-media.ts

```diff
@@ -44,12 +45,25 @@ export class HaMediaSelector extends LitElement {
       if (thumbnail === oldThumbnail) {
         return;
       }
-      if (thumbnail && thumbnail.startsWith("/")) {
+      if (thumbnail) {
         this._thumbnailUrl = undefined;
-        // Thumbnails served by local API require authentication
-        getSignedPath(this.hass, thumbnail).then((signedPath) => {
-          this._thumbnailUrl = signedPath.path;
-        });
+        if (thumbnail.startsWith("/")) {
+          // Thumbnails served by local API require authentication
+          getSignedPath(this.hass, thumbnail).then((signedPath) => {
+            this._thumbnailUrl = signedPath.path;
+          });
+        }
+
+        if (thumbnail.startsWith("https://brands.home-assistant.io")) {
```

Comment:
This is wrong. Now if thumbnail doesn't start with `/` or brands url, it will never be set to `this._thumbnailUrl` because you've changed the first if-statement to check if thumbnail is truthy, and so it handles all cases.